guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -16,18 +16,21 @@ For more information, see {ReleaseNotesDocURL}[Release notes].
 endif::[]
 * {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
 ifdef::satellite[]
-* For {SmartProxy} upgrade, enable and synchronize the following repositories to {ProjectServer}:
-+
-Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs):: `{RepoRHEL9BaseOS}`
-+
-Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs):: `{RepoRHEL9AppStream}`
-+
-Red Hat Satellite Capsule {ProjectVersion} for RHEL 9 x86_64 RPMs:: `{RepoRHEL9ServerSatelliteCapsuleProjectVersion}`
-+
-Red Hat Satellite Maintenance {ProjectVersion} for RHEL 9 x86_64 RPMs:: `{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}`
-
-+
-Add the repositories to the lifecycle environment and content view that is attached to your {SmartProxyServer}.
+* If you are upgrading a {SmartProxy}, enable and synchronize the following repositories to {ProjectServer}, and add them to the lifecycle environment and content view that is attached to your {SmartProxyServer}:
+** *Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)*:
+*** `{RepoRHEL9BaseOS} 9`
```

Comment:
`RepoRHEL9BaseOS` is the repo label (`rhel-9-for-x86_64-baseos-rpms`) which does *not* contain the `9` or `9.Y` anywhere, so I'd avoid using it in this line.

Maybe:
```
** *Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)* (`{RepoRHEL9BaseOS}`):
*** 9
*** 9._Y_
```

idk, CDN structure is hard

Reply:
Noted, I tried something different with the latest commit.